docs/setup.md

```diff
@@ -1,5 +1,5 @@
 ---
-nav_order: 2
+nav_order: 3
```

Comment:
This looked incorrect. The `frontend.md` file also used `nav_order: 2`, and the `testing.md` which appears next in the docs had `nav_order: 4`